src/sonic-yang-models/setup.py

```diff
@@ -50,6 +50,10 @@
                          './yang-models/sonic-flex_counter.yang',
                          './yang-models/sonic-interface.yang',
                          './yang-models/sonic-loopback-interface.yang',
+                         './yang-models/sonic-mgmt-interface.yang',
+                         './yang-models/sonic-mgmt-port.yang',
```

Comment:
sonic-mgmt-port ---> sonic-mgmt_port

Reply:
Done.

Reply:
We already have "sonic-loopback-interface.yang". Why the need for this change?